include/asterisk/channel.h

```diff
@@ -2919,6 +2924,19 @@ struct ast_group_info {
 	AST_LIST_ENTRY(ast_group_info) group_list;
 };
 
+/*! \brief list of groups currently in use, with a pointer to a list of channels within the groups
+  */
```

Comment:
The comment close could just be at the end of the preceding line here

Reply:
Fixed